src/pages/steps/ScopeStep.tsx

```diff
@@ -1,19 +1,33 @@
 import { useRouter } from 'next/router';
 import { useEffect } from 'react';
+import { Alert } from 'react-bootstrap';
```

Comment:
I think you can use the `Alert` from our UI-library (see `src/ui/Alert`) instead of the bootstrap one.